k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -182,8 +188,8 @@ void stop() throws InterruptedException {
         controllerThread.join(1000L);
     }
 
-    @Override
-    public void run() {
+    @WorkerThread
+    private void runInBackground() {
```

Comment:
Adding the `@WorkerThread` annotation here is adding more noise than useful information. Try to avoid the thread-related annotations on methods where it's obvious on which thread they're called.